rsmtool/utils.py

```diff
@@ -123,17 +157,32 @@ def agreement(score1, score2, tolerance=0):
     return (float(num_agreements) / len(score1)) * 100
```

Comment:
Elsewhere these were changed to
agreement = float(num_agreements)....
return agreement